### DISCUSSION TITLE
HEIC format

### DISCUSSION THREAD

**Discussion by treblechet** [d2phap/ImageGlass#768] [General] [answered]
### Is your feature request related to a problem? Please describe.

In Windows 10 (1909) and ImageGlass (7.6.4.30) I do not see the image thumbnails for .HEIC files, only the ImageGlass thumbnail. I was wondering if this would be implemented soon (or if it is, how to enable it)
### Describe the solution you'd like

I would like there to be image thumbnails visible for .HEIC files in the same way as they're currently implemented for other image formats.
### Describe alternatives you've considered

I would like to mention that if it is the case that this feature is already present and I am not configured properly to see it, I would also like help on that. My settings seem to be configured properly to see thumbnails of images, just not of .HEIC files
### Additional context

For a screenshot of what I mean, I posted one below
![image](https://user-images.githubusercontent.com/50604280/80774230-66ee4500-8b2a-11ea-8cef-03b5f19da421.png)

**Comment by d2phap** [comment, marked as answer]
You can download this extension from Store: https://www.microsoft.com/en-us/p/heif-image-extensions/9pmmsr1cgpwg?activetab=pivot:overviewtab

**Reply by treblechet** [reply]
So to my understanding, thumbnails are rendered and created by Windows, not the program. If that is true, and it cannot be changed, then it would make sense that I could not load .HEIC thumbnails. I believe that most images are supported by the OS itself, but HEIC images are not, and are supported by a plug-in to Microsoft Photos. When I had removed that (and therefore the plugin as well) it would only produce thumbnails for images that Windows could process natively. 

If this is true and someone can confirm this, then I will mark this as answered, but if it possible to have ImageGlass doing this, it would allow me to completely remove Microsoft Photos as I don't need two photo viewing apps on my computer.

Edit: I should also mention, as a small bit of knowledge to those who may be concerned, programs that would manipulate .HEIC images will not recognize them as images without Microsoft Photos/the .HEIC plugin. Installing those two again allowed me to use XnConvert for the .HEIC images

**Reply by d2phap** [reply]
I think you just need to install HEIC extension.
Honestly, I haven't tried remove MS Photos before.
